dist/war/getstarted/index.html

```diff
@@ -78,6 +77,26 @@ <h3 class="h4">Resource manager</h3>
         </div>
       </div>
 
+      <h4 class="mt-5">APIs (REST, GraphQL, Javadoc)</h4>
+      <p>
+        <a id="restUrlLink" href="/doc/rest/" target=”_blank”
+           title="Scheduler REST API Documentation">Scheduler REST API</a>,
+        <a id="restGraphQLLink" href="/scheduling-api/v1/graphiql" target=”_blank”
```

Comment:
I think it is more readable to put abreviation in minor case such as Url. => replace by restGraphQlLink